src/Module/ui.js

```diff
@@ -55,25 +60,24 @@ const display = () => {
           listData[index].completed = true;
           inputTexts[index].disabled = true;
           inputTexts[index].previousElementSibling.disabled = true;
-          localStorage.setItem('tasks', JSON.stringify(listData));
+          saveData(listData);
         });
       });
 
-      trashBtns.forEach((trashBtn, index) => {
-        trashBtn.addEventListener('click', () => {
-          const indexRem = listData.findIndex(
-            (task) => task.description === inputTexts[index].textContent,
-          );
-          listData.splice(indexRem, 1);
-          listData.forEach((taskDay, indexDay) => {
-            taskDay.index = indexDay + 1;
-          });
-          localStorage.setItem('tasks', JSON.stringify(listData));
-          trashBtn.parentNode.parentNode.remove();
-        });
-        trashBtns = document.querySelectorAll('.trash-btn');
-      });
+      // trashBtns.forEach((trashBtn, index) => {
+      //   trashBtn.addEventListener('click', () => {
+      //     // const indexRem = listData.findIndex(
+      //     //   (task) => task.description === inputTexts[index].textContent
+      //     // );
+      //     // listData.splice(indexRem, 1);
+      //     // listData.forEach((taskDay, indexDay) => {
+      //     //   taskDay.index = indexDay + 1;
+      //     // });
+      //     // saveData(listData);
+      //     // trashBtn.parentNode.parentNode.remove();
+      //   });
+      //   trashBtns = document.querySelectorAll('.trash-btn');
+      // });
```

Comment:
- [ ] [optional] Please remove commented code from the codebase as mentioned in the best practices guidelines **'Do not commit old pieces of code as inline comments. They will make your project look messy.'**